tests/adapters/compliance_tests/get_bond.py

```diff
@@ -0,0 +1,21 @@
+from hamcrest import assert_that, equal_to
+from netman.core.objects.exceptions import UnknownBond
+from tests.adapters.compliance_test_case import ComplianceTestCase
+
+
+class GetBondTest(ComplianceTestCase):
+    _dev_sample = "juniper"
+
+    def test_get_bond_and_get_bonds_return_the_same(self):
+        self.client.add_bond(42)
+
+        bond_from_get_bond = self.client.get_bond(42)
```

Comment:
you should try to set all the possible values here, or at least some, like

https://github.com/internap/netman/blob/master/tests/adapters/compliance_tests/get_vlan_test.py